wizeline/falcon/middlewares/bodyParser.py

```diff
@@ -0,0 +1,58 @@
+import json
+from json.decoder import JSONDecodeError
+
+import falcon
+
+class BodyParserMiddleware:
+    def process_resource(self, req, resp, resource, params):
+        if self._is_middleware_enabled(resource) and self._request_method_has_payload(req):
+            if self._is_json_content_type(req):
+                try:
+                    req.text = self._get_payload(req)
+
+                    if req.text.strip() != '':
+                        req.json = json.loads(req.text)
+                    else:
+                        req.json = {}
+
+                except JSONDecodeError:
+                    raise falcon.HTTPInternalServerError()
```

Comment:
I think is strange to raise a `HTTP 500` error when sending an invalid JSON to a bodyParser MIddleware